Data-Science/Kubeflow/MNIST-Digits-Recognition/pvc.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-name
+spec:
+  accessModes:
+   - ReadWriteOnce
+  resources:
+   requests:
+    storage: 1Gi
```

Comment:
Please add a newline at the end of file.

Reply:
Done